enterprise/cmd/frontend/internal/auth/githuboauth/session.go

```
@@ -120,6 +125,30 @@ func (s *sessionIssuerHelper) GetOrCreateUser(ctx context.Context, token *oauth2
 			CreateIfNotExist:    attempt.createIfNotExist,
 		})
 		if err == nil {
+			for _, installation := range installations {
+				accountID := strconv.FormatInt(*installation.ID, 10) + "/" + strconv.FormatInt(derefInt64(ghUser.ID), 10)
+				_, _, err := auth.GetAndSaveUser(ctx, s.db, auth.GetAndSaveUserOp{
+					UserProps: database.NewUser{
+						Username:        login,
+						Email:           attempt.email,
+						EmailIsVerified: true,
+						DisplayName:     deref(ghUser.Name),
+						AvatarURL:       deref(ghUser.AvatarURL),
+					},
+					ExternalAccount: extsvc.AccountSpec{
+						ServiceType: fmt.Sprintf("%sApp", s.ServiceType),
+						ServiceID:   s.ServiceID,
+						ClientID:    s.clientID,
+						AccountID:   accountID,
+					},
+					CreateIfNotExist: attempt.createIfNotExist,
+				})
+
+				if err != nil {
+					fmt.Println("ERROR: ", err)
```

Should we log this error? Or return it? Instead of swallowing and printing to stdout?

---

Oh I just hard missed a Println, that was not on purpose